documentation/source/user_section/installation/windows.rst

```diff
@@ -4,60 +4,82 @@
-        If you are comfortable with installing SCT with WSL2, please feel free to do so.
+   - After this command finishes, you will be prompted to restart your computer.
 
-        When asked what Linux version to install, select the Ubuntu 18.04 LTS distro.
+   - After restarting, the installation should automatically resume, and you will be able to create a user account inside Ubuntu by selecting a username and password.
+
+#. Choose the WSL version (1/2).
+
+   By default, Microsoft's instructions will create an Ubuntu environment using Version 2 of WSL. While version 2 has been tested to work with SCT, our development team tests more thoroughly using Version 1 of WSL (due to `better support from GitHub Actions <https://github.com/actions/virtual-environments/issues/50>`_).
+
+   Because of this, we recommend that you convert the WSL2 Ubuntu environment to use WSL1 before continuing. To do this, first close Ubuntu, then re-open Powershell and run the following command:
+
+   .. code::
+
+      wsl --list --verbose
+
+   If WSL installed correctly, you should see a list of installed distributions (as well as their WSL versions). Find the name of the distribution you just installed (which should be something like ``Ubuntu`` or ``Ubuntu-20.04``), then run the following command:
+
+   .. code::
+
+      wsl --set-version Ubuntu 1
+
+   You can then run ``wsl --list --verbose`` again to check that the distribution has changed from WSL2 to WSL1.
 
 #. Environment preparation
 
-   Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
+   Now that you have set up an Ubuntu environment with WSL, please open Ubuntu and run the following command to install various packages that will be needed to install SCT.
 
    .. code-block:: sh
 
-      sudo apt-get update
-      sudo apt-get -y install gcc
-      sudo apt-get -y install unzip
-      sudo apt-get install -y python-pip python
-      sudo apt-get install -y psmisc net-tools
-      sudo apt-get install -y git
-      sudo apt-get install -y gfortran
-      sudo apt-get install -y libjpeg-dev
-      echo 'export DISPLAY=127.0.0.1:0.0' >> ~/.profile
+      sudo apt-get update && sudo apt-get upgrade -y
+      sudo apt-get -y install gcc unzip python3-pip python3 psmisc net-tools git gfortran libjpeg-dev
 
 #. Install SCT
 
-   Download SCT:
+   First, download SCT by running the following commands in Ubuntu:
 
    .. code-block:: sh
 
-      git clone https://github.com/spinalcordtoolbox/spinalcordtoolbox.git sct
-      cd sct
+      cd ~
+      git clone https://github.com/spinalcordtoolbox/spinalcordtoolbox.git
+      cd spinalcordtoolbox
 
    To select a `specific release <https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases>`_, replace X.Y.Z below with the proper release number. If you prefer to use the development version, you can skip this step.
```

Comment:
I know it's not part of this PR, but do we want to remove the "Z" now that we decided to remove patches? (and maybe give an example because some people might not understand what we mean by X.Y., eg: 5.4)